src/rtScriptDuk/rtScriptDuk.cpp

```diff
@@ -1058,7 +1059,7 @@ rtError rtDukContext::runFile(const char *file, rtValue* retVal /*= NULL*/, cons
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-rtScriptDuk::rtScriptDuk():mRefCount(0)
+rtScriptDuk::rtScriptDuk():mRefCount(0), node_is_initialized(false)
 #ifndef RUNINMAIN
```

Comment:
We should probably find another spot to initialize node_is_initialized to false.  Somewhere outside of the duktape sources.